### PR TITLE
Fix wrong image on classical kernels tutorial

### PR DESCRIPTION
**Pull request by emgilfuster**
------------------------------------------------------------------------------------------------------------

**Title:** Fix wrong image on classical kernels tutorial

**Summary:** 
The [classical kernels tutorial](https://pennylane.ai/qml/demos/tutorial_classical_kernels.html) has the wrong image (from a different demo). The files are both called `flowchart.png` and this is suspected to be the problem. So the only thing I did was renaming the conflicting file to `classical_kernels_flowchart.png` and changing the one line in `demonstrations/tutorial_classical_kernels.py`.

**Relevant references:**
The classical kernels tutorial [https://pennylane.ai/qml/demos/tutorial_classical_kernels.html](https://pennylane.ai/qml/demos/tutorial_classical_kernels.html)
The existing image that I renamed [https://github.com/PennyLaneAI/qml/blob/master/demonstrations/classical_kernels/flowchart.PNG](https://github.com/PennyLaneAI/qml/blob/master/demonstrations/classical_kernels/flowchart.PNG).
The existing file where I changed one line [https://github.com/PennyLaneAI/qml/blob/master/demonstrations/tutorial_classical_kernels.py](https://github.com/PennyLaneAI/qml/blob/master/demonstrations/tutorial_classical_kernels.py)
In case it adds something, the image that appears right now in the demo, which belongs to a different demo [https://github.com/PennyLaneAI/qml/blob/master/demonstrations/quantum_analytic_descent/flowchart.png](https://github.com/PennyLaneAI/qml/blob/master/demonstrations/quantum_analytic_descent/flowchart.png)

**Possible Drawbacks:** Hopefully none

**Related GitHub Issues:** #608

### CODE REVIEW

**Comment by KetpuntoG** [comment]
I think the mistake comes from our side
I'll take care of it. I'll let you know if it works :)

**Comment by KetpuntoG** [comment]
oh I have seen it @emgilfuster , what happens is that the image is the one used also in the `demo_qml.rst` file, you must change the image name there also

**Comment by emgilfuster** [comment]
Ah, of course! I forgot about that one, oopsie!
The checks are running and I'm signing off for today, we'll see how it goes!
Thanks for the help, you made it much easier :)

File: demos_qml.rst [hunk 1]
```diff
@@ -100,7 +100,7 @@ IBM, Google, Rigetti, and many more—or on real hardware devices.
 
 .. gallery-item::
     :tooltip: Estimate a classical kernel function on a quantum computer.
-    :figure: demonstrations/classical_kernels/flowchart.PNG
+    :figure: demonstrations/classical_kernels/classical_kernel_sflowchart.PNG
```
Comment (josh146) [inline demos_qml.rst:103]:
```suggestion
    :figure: demonstrations/classical_kernels/classical_kernels_flowchart.PNG
```

**Comment by emgilfuster** [comment]
thanks @josh146, that was quite an embarrassing typo.
The built version looks good to me!
Thanks for all the help :)

**Review by KetpuntoG — APPROVED** [review]
Great! Thanks for fixing this 💪

**Pull request merged**